python/taichi/__init__.py

```diff
@@ -11,6 +11,8 @@
 # Issue#2223: Do not reorder, or we're busted with partially initialized module
 from taichi import aot  # isort:skip
 
+from taichi.ui import *  # isort:skip
```

Comment:
nit: Let's not do this now

Reply:
Are you referring to the * or the isort:skip or the import itself?

Reply:
`import *`. It's fine to use `ti.ui`

Reply:
Ah of course. This is not what I intended to do 😅 will fix.